cmake/Modules/FindOnnxRuntime.cmake

```diff
@@ -0,0 +1,18 @@
+find_library(ONNXRUNTIME_LIB 
+  NAMES onnxruntime
+  )
+find_path(ONNXRUNTIME_INCLUDE_DIR_ROOT NAMES include/onnxruntime/core/session)
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(onnxruntime
+  DEFAULT_MSG ONNXRUNTIME_LIB ONNXRUNTIME_INCLUDE_DIR_ROOT)
+
+if(ONNXRUNTIME_FOUND)
+  add_library(onnxruntime SHARED IMPORTED)
+  set_target_properties(onnxruntime PROPERTIES IMPORTED_LOCATION ${ONNXRUNTIME_LIB})
+
+  set(ONNXRUNTIME_INCLUDE_DIRS ${ONNXRUNTIME_INCLUDE_DIR_ROOT}/include/onnxruntime)
```

Comment:
The preferred way of doing this is setting the include dirs on the onnxruntime target itself